bounceview/src/main/java/com/github/hariprasanths/bounceview/CustomDialog.java

```diff
@@ -0,0 +1,243 @@
+package com.github.hariprasanths.bounceview;
+
+import ohos.agp.components.Button;
+import ohos.agp.components.Component;
+import ohos.agp.components.ComponentContainer;
+import ohos.agp.components.DirectionalLayout;
+import ohos.agp.components.Image;
+import ohos.agp.components.LayoutScatter;
+import ohos.agp.components.Text;
+import ohos.agp.window.dialog.CommonDialog;
+import ohos.app.Context;
+
+/**
+ * Custom Common Dialog.
+ */
+public class CustomDialog extends CommonDialog {
+    Component baseCustomLayout;
```

Comment:
rename this variable to baseDialogLayout

Reply:
done